prow/gerrit/adapter/adapter.go

```diff
@@ -397,7 +397,8 @@ func (c *Controller) handleInRepoConfigError(err error, instance string, change
 	if err != nil {
 		// If we have not already recorded this failure send an error essage
 		if failed, ok := c.inRepoConfigFailures[key]; !ok || !failed {
-			if setReviewWerr := c.gc.SetReview(instance, change.ID, change.CurrentRevision, inRepoConfigFailed, nil); setReviewWerr != nil {
+			msg := fmt.Sprintf("%s: %s", err, inRepoConfigFailed)
```

Comment:
```suggestion
			msg := fmt.Sprintf("%v: %s", err, inRepoConfigFailed)
```
otherwise it will throw at runtime I think